vnext/Microsoft.ReactNative/ABIViewManager.cpp

```diff
@@ -53,9 +53,15 @@ const wchar_t *ABIViewManager::GetName() const {
   return m_name.c_str();
 }
 
-xaml::DependencyObject ABIViewManager::CreateViewCore(int64_t) {
-  auto view = m_viewManager.CreateView();
-  return view;
+xaml::DependencyObject ABIViewManager::CreateViewCore(
+    int64_t,
+    const winrt::Microsoft::ReactNative::JSValueObject &props) {
+  if (auto viewCreateProps = m_viewManager.try_as<IViewManagerCreateWithProperties>()) {
+    auto view = viewCreateProps.CreateViewWithProperties(
+        MakeJSValueTreeReader(winrt::Microsoft::ReactNative::JSValue(props.Copy())));
+    return view.as<xaml::DependencyObject>();
```

Comment:
Don't need the cast if it the interface returned a DO to start with.

Reply:
see the other comment about long term being "bilingual"